source/getting-started/configuration.markdown

```diff
@@ -41,8 +41,8 @@ Now let's make a change using the file editor: we are going to change the name,
 </div>
 
  - Click the save icon in the top right to commit changes.
- - Most changes in `configuration.yaml` require Home Assistant to be restarted to see the changes. You can verify that your changes are acceptable by running a configuration check. Do this by clicking on Configuration in the sidebar, click on `Developer-Tools -> YAML` and click on the **Check configuration** button. When it's valid, it will show the text "Configuration valid!". In order for the **Check Configuration**" button to be visible, you must enable "Advanced Mode" on your user profile.
- - Now Restart Home Assistant using the **Restart** button in the Server management section on the same page.
+ - Most changes in `configuration.yaml` require Home Assistant to be restarted to see the changes. You can verify that your changes are acceptable by running a configuration check. Do this by navigating to {% my server_controls title="Developer-Tools -> YAML" %} and and then clicking on the **Check configuration** button. When it's valid, it will show the text "Configuration valid!". In order for the **Check Configuration**" button to be visible, you must enable "Advanced Mode" on your user profile.
+ - Now Restart Home Assistant. You can do so by either using the **Restart HASS** option in the ⚙ menu of the File Editor UI, or by navigating to {% my system_dashboard title="Settings -> System" %} and then clicking on the **Restart** button on the top right of the page.
```

Comment:
We don't allow the use of the term `HASS` in our documentation.

```suggestion
 - Now Restart Home Assistant. You can do so by either using the **Restart** option in the ⚙ menu of the File Editor UI or by navigating to {% my system_dashboard title="Settings -> System" %} and then clicking on the **Restart** button on the top right of the page.
```

Reply:
Ok. But shouldn't the actual option text in the File Editor add-on be changed in the first place?

Reply:
The file editor itself is not created or maintained by the Home Assistant project.

Reply:
I'm a bit confused. The source repo of the add-on container seems to be under the `home-assistant` GH org: https://github.com/home-assistant/addons/tree/master/configurator

But I couldn't locate any strings of the ⚙ menu there... where does that code live?

Reply:
Add-on wrap existing applications, in case of the File editor add-on is packages the HASS-Configurator: <https://github.com/danielperna84/hass-configurator>

Reply:
I tried... 🤞: https://github.com/danielperna84/hass-configurator/pull/222